daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SExpr.scala

```diff
@@ -222,12 +211,9 @@ object SExpr {
     */
   sealed abstract class SELoc extends SExprAtomic {
     def lookup(machine: Machine): SValue
-    def evaluate(machine: Machine): SValue = {
+    def lookupValue(machine: Machine): SValue = {
```

Comment:
Why do we have to separate names for the same thing now?

Reply:
I removed the now unnecessary `lookup`